libraries/AP_HAL/board/px4.h

```diff
@@ -95,7 +95,7 @@
 #define HAL_BARO_MS5607_I2C_ADDR 0x77
 #define HAL_BARO_MS5607_I2C_BUS  0
 #define HAL_COMPASS_IST8310_I2C_ADDR 0x0E
-#define HAL_COMPASS_IST8310_I2C_BUS 1
+#define HAL_COMPASS_IST8310_I2C_BUS 2
```

Comment:
Looks like I did not update it with the changes to keep IST8310 in the same I2C bus and just change the ADC to another bus.